2025-07-06 04:12:15.691 - RealTimeSTT: root - INFO - Starting RealTimeSTT
2025-07-06 04:12:15.709 - RealTimeSTT: root - INFO - Initializing audio recording (creating pyAudio input stream, sample rate: 16000 buffer size: 512
2025-07-06 04:12:15.718 - RealTimeSTT: root - INFO - Initializing WebRTC voice with Sensitivity 3
2025-07-06 04:12:15.720 - RealTimeSTT: root - DEBUG - WebRTC VAD voice activity detection engine initialized successfully
2025-07-06 04:12:17.262 - RealTimeSTT: torio._extension.utils - DEBUG - Loading FFmpeg6
2025-07-06 04:12:17.276 - RealTimeSTT: torio._extension.utils - DEBUG - Failed to load FFmpeg6 extension.
Traceback (most recent call last):
  File "C:\Users\hp\miniconda3\lib\site-packages\torio\_extension\utils.py", line 116, in _find_ffmpeg_extension
    ext = _find_versionsed_ffmpeg_extension(ffmpeg_ver)
  File "C:\Users\hp\miniconda3\lib\site-packages\torio\_extension\utils.py", line 108, in _find_versionsed_ffmpeg_extension
    _load_lib(lib)
  File "C:\Users\hp\miniconda3\lib\site-packages\torio\_extension\utils.py", line 94, in _load_lib
    torch.ops.load_library(path)
  File "C:\Users\hp\miniconda3\lib\site-packages\torch\_ops.py", line 1032, in load_library
    ctypes.CDLL(path)
  File "C:\Users\hp\miniconda3\lib\ctypes\__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'C:\Users\hp\miniconda3\Lib\site-packages\torio\lib\libtorio_ffmpeg6.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
2025-07-06 04:12:17.281 - RealTimeSTT: torio._extension.utils - DEBUG - Loading FFmpeg5
2025-07-06 04:12:17.290 - RealTimeSTT: torio._extension.utils - DEBUG - Failed to load FFmpeg5 extension.
Traceback (most recent call last):
  File "C:\Users\hp\miniconda3\lib\site-packages\torio\_extension\utils.py", line 116, in _find_ffmpeg_extension
    ext = _find_versionsed_ffmpeg_extension(ffmpeg_ver)
  File "C:\Users\hp\miniconda3\lib\site-packages\torio\_extension\utils.py", line 108, in _find_versionsed_ffmpeg_extension
    _load_lib(lib)
  File "C:\Users\hp\miniconda3\lib\site-packages\torio\_extension\utils.py", line 94, in _load_lib
    torch.ops.load_library(path)
  File "C:\Users\hp\miniconda3\lib\site-packages\torch\_ops.py", line 1032, in load_library
    ctypes.CDLL(path)
  File "C:\Users\hp\miniconda3\lib\ctypes\__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'C:\Users\hp\miniconda3\Lib\site-packages\torio\lib\libtorio_ffmpeg5.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
2025-07-06 04:12:17.290 - RealTimeSTT: torio._extension.utils - DEBUG - Loading FFmpeg4
2025-07-06 04:12:17.297 - RealTimeSTT: torio._extension.utils - DEBUG - Failed to load FFmpeg4 extension.
Traceback (most recent call last):
  File "C:\Users\hp\miniconda3\lib\site-packages\torio\_extension\utils.py", line 116, in _find_ffmpeg_extension
    ext = _find_versionsed_ffmpeg_extension(ffmpeg_ver)
  File "C:\Users\hp\miniconda3\lib\site-packages\torio\_extension\utils.py", line 108, in _find_versionsed_ffmpeg_extension
    _load_lib(lib)
  File "C:\Users\hp\miniconda3\lib\site-packages\torio\_extension\utils.py", line 94, in _load_lib
    torch.ops.load_library(path)
  File "C:\Users\hp\miniconda3\lib\site-packages\torch\_ops.py", line 1032, in load_library
    ctypes.CDLL(path)
  File "C:\Users\hp\miniconda3\lib\ctypes\__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'C:\Users\hp\miniconda3\Lib\site-packages\torio\lib\libtorio_ffmpeg4.pyd' (or one of its dependencies). Try using the full path with constructor syntax.
2025-07-06 04:12:17.299 - RealTimeSTT: torio._extension.utils - DEBUG - Loading FFmpeg
2025-07-06 04:12:17.299 - RealTimeSTT: torio._extension.utils - DEBUG - Failed to load FFmpeg extension.
Traceback (most recent call last):
  File "C:\Users\hp\miniconda3\lib\site-packages\torio\_extension\utils.py", line 116, in _find_ffmpeg_extension
    ext = _find_versionsed_ffmpeg_extension(ffmpeg_ver)
  File "C:\Users\hp\miniconda3\lib\site-packages\torio\_extension\utils.py", line 106, in _find_versionsed_ffmpeg_extension
    raise RuntimeError(f"FFmpeg{version} extension is not available.")
RuntimeError: FFmpeg extension is not available.
2025-07-06 04:12:17.977 - RealTimeSTT: root - DEBUG - Silero VAD voice activity detection engine initialized successfully
2025-07-06 04:12:17.983 - RealTimeSTT: root - DEBUG - Starting realtime worker
2025-07-06 04:12:17.983 - RealTimeSTT: root - DEBUG - Waiting for main transcription model to start
